packages/rocketchat-api/server/api.js

```diff
@@ -34,7 +34,7 @@ class API extends Restivus {
 				if (RocketChat.settings.get('API_Enable_CORS') === true) {
 					this.response.writeHead(200, {
 						'Access-Control-Allow-Origin': RocketChat.settings.get('API_CORS_Origin'),
-						'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept, X-User-Id, X-Auth-Token'
+						'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept, X-User-Id, X-Auth-Token, Authorization: Bearer'
```

Comment:
Sorry, but I fail to see how this will fix anything. ` Bearer` is part of the `Authorization` header's value. So, removing that would maybe make it work...maybe I am wrong though. What are your thoughts? Have you tried both ways?

Reply:
I have outlined my idea behind and my tests behind this fix in the original issue: https://github.com/RocketChat/Rocket.Chat/issues/8292#issuecomment-332585692

Reply:
And I understand that, however for a header's key to be properly name it must not contain any spaces or symbols other than a hyphen. Thus, having `Authorization: Bearer` will be invalid and instead changing it to just `Authorization` will have the desired effect. For more information, please see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization

Reply:
Thank you very much for your feedback. I looked a bit around and you are right, it seems that Authorization is the correct "header name". This [spec](https://fetch.spec.whatwg.org/#cors-non-wildcard-request-header-name) helped me understanding it.